packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.html

```diff
@@ -5,26 +5,21 @@
 <div data-ui-view>
     <div class="datacenter-backup">
         <div data-ovh-alert="{{ $ctrl.alertMain }}"></div>
-        <oui-message
-            class="mb-3"
-            data-ng-if="!$ctrl.backup.isActive()"
-            data-type="info"
-        >
-            <span
-                data-translate="{{ 'dedicatedCloud_datacenter_backup_state_' + $ctrl.backup.state }}"
-            ></span>
-            <p
-                data-translate="dedicatedCloud_datacenter_backup_not_manageable"
-            ></p>
-        </oui-message>
         <div data-ng-if="$ctrl.backup.isActive()">
-            <oui-switch
-                data-model="$ctrl.backup.isActive()"
-                data-on-change="$ctrl.goToDeleteBackup()"
-            >
-            </oui-switch>
+            <div class="oui-switch">
+                <label class="oui-switch__label-container">
+                    <input
+                        type="checkbox"
+                        class="oui-switch__checkbox"
+                        checked="checked"
+                        onchange="this.checked=true"
```

Comment:
Native events won't be digested by angular ;)
```suggestion
                        ng-change="this.checked=true"
```

Reply:
The requirement is that the switch should appear enabled always and we should not be able to switch it off. When the user clicks on it, we display a pop-up to deactivate the backup. The above code just sets the ui state of the checkbox to checked (and does not change the model). So it is not in angular scope. The `this.checked` is the native checkbox's checked property. There is no model associated with this control and we just need the on-click to trigger the pop-up